### ISSUE TITLE
don't replace text in original note + include link in created note

### ISSUE THREAD

**Issue by mendelkranz** [lynchjames/note-refactor-obsidian#25]
This is really a two part request, both of which I imagine would be options rather than default functionality:

1. do _not_ replace the text in the initial note with a link, and just leave it as is
2. add a link in the newly created note to the original note 

use case: I take literature notes where I jot things down next to quotes. sometimes I'll want to make a longer note about a particular idea, but want to leave the literature note as is. (perhaps a niche use case, but I can imagine other scenarios where this would be helpful).

Thanks for creating new functionality and a great plugin!

**Comment by lynchjames** [comment]
This may be possible with a template placeholder `{{new_note_content}}` which could be introduced along with #24.

**Comment by lynchjames** [comment]
Available in version 1.5.0.

**Comment by mendelkranz** [comment]
Thanks for the update! However, I'd like to open this back up again, if I may, specifically with regard to issue number 1 mentioned above:

> 1. do not replace the text in the initial note with a link, and just leave it as is

I'm not sure the problem was solved, unless I am misunderstanding something. There still doesn't seem to be a way to leave the original note/text as is when refactoring to a new note. 

I tried using the {{new_note_content}} and placing that in the 'Note link template,' however that includes _everything_ in the new note (including whatever titles and templates I include there, so I am still left without a way to simply leave the original text as is in the original file.
